components/lib/hooks/useStyle.js

```diff
@@ -21,7 +21,7 @@ export const useStyle = (css, options = {}) => {
 
         const styleContainer = context?.styleContainer || document.head;
 
-        styleRef.current = styleContainer.querySelector(`style[data-primereact-style-id="${name}"]`) || document.getElementById(id) || document.createElement('style');
+        styleRef.current = styleContainer.querySelector(`style[data-primereact-style-id="${name}"]`) || document.querySelector(`#${id}`) || document.createElement('style');
```

Comment:
This change won't fixe the issue. New selector `document.querySelector('#undefined')` will select the same element with `id="undefined"`.. :(

Reply:
I belive, you something like that:
```js
const getCurrentStyleRef = () => {
  const existingStyleContainerChild = styleContainer.querySelector(
    `style[data-primereact-style-id="${name}"]`,
  );
  if (existingStyleContainerChild) {
    return existingStyleContainerChild;
  }
  // First, check that id is defined!
  if (id !== undefined) {
    // And select element by defined id on the page after:
    const existingElementWithId = document.getElementById(id);
    if (existingElementWithId) {
      return existingElementWithId;
    }
  }
  return document.createElement('style');
};
styleRef.current = getCurrentStyleRef();
```

Reply:
Yes, I made a mistake, sorry. I forgot add during the testing with #